libraries/chain/include/graphene/chain/protocol/asset_ops.hpp

```diff
@@ -442,11 +442,42 @@ namespace graphene { namespace chain {
       void            validate()const;
    };
 
+   /**
+    * @brief Transfers BTS from the fee pool of a specified asset back to the issuer's balance
+
+    * @param fee Payment for the operation execution
+    * @param issuer Account which will be used for transfering BTS
+    * @param asset_id Id of the asset whose fee pool is going to be drained
+    * @param amount_to_claim Amount of BTS to claim from the fee pool
+    * @param extensions Field for future expansion
+
+    * @pre @ref fee must be paid in the asset other than the one whose pool is being drained
+    * @pre @ref amount_to_claim should be specified in the core asset
+    * @pre @ref amount_to_claim should be nonnegative
+    */
+   struct asset_claim_pool_operation : public base_operation
+   {
+      struct fee_parameters_type {
+         uint64_t fee = 20 * GRAPHENE_BLOCKCHAIN_PRECISION;
```

Comment:
The default fee is perhaps too high. Although it can be changed by the committee later, I think it's better be handled like `bid_collateral_operation` ([code](https://github.com/bitshares/bitshares-core/blob/2.0.171105a/libraries/chain/include/graphene/chain/protocol/fee_schedule.hpp#L65-L79)).

Reply:
This was interesting. I added the code as requested. I assume the code does the following: Figure out if the blockchain parameters have a fee for that operation already, or otherwise pick another fee for it instead. In my case, i chose the fee for `asset_fund_fee_pool_operation`